internal/http/services/owncloud/ocdav/dav.go

```diff
@@ -52,6 +52,14 @@ type DavHandler struct {
 	OCMSharesHandler    *WebDavHandler
 }
 
+const (
```

Comment:
Some of these errors are not being used... Wether we add them somewhere where they make sense (the best) or we remove them (not so "best").